src/Register.js

```diff
@@ -74,31 +74,37 @@ class Register extends React.Component {
   render() {
     return (
       <div className="Register">
-          <header> Register for PP Friends</header>
+          <h1> Join the Community</h1>
```

Comment:
hehe i like this! i think it attracts people to register for the app because it makes them feel like they are apart of something cool/interesting/unique